### DISCUSSION TITLE
When will the difference between main and 1.2.x be reflected in the version?

### DISCUSSION THREAD

**Discussion by kyaryunha** [mido/mido#482] [Q&A] [answered]
I read mido github because I think I need a function to return int to second2tick.

Looking at the main branch, there was something that superbock had already coded in 2018.

<img width="50%" src="https://github.com/mido/mido/assets/37409087/2c26f03a-eb74-4eec-bec1-759a95aa6ff1"/>

However, in the latest version of mido 1.12., it still returns as a float.

Looking at the branch, even though it was released in 2021, the content merged in 2018 has not yet been reflected.
https://github.com/mido/mido/compare/1.2.x...main 
<img width="50%" src="https://github.com/mido/mido/assets/37409087/29098297-bca0-4e8d-a4af-28616affee66"/>

I wonder when this feature will be deployed.
I wonder if you forgot to deploy it yet or if it is being put on hold for some other reason.
If it's on hold for some other reason, I wonder why.

**Comment by rdoursenaud** [comment, marked as answer]
Hi and thank you for using MIDO,

The short answer is: the proposed commits were from 2018 but we only merged them this year including a fix.
See Pull Request #454 (Original was #114).

These changes will be in the upcoming release that is due by the end of July if everything goes according to plan: https://github.com/mido/mido/milestone/1

P.S.: Please refrain from using issues for questions and prefer the new discussions feature instead. I've made the move for you this time. Thanks.

**Comment by rdoursenaud** [comment]
Version 1.3.0 has been released including the requested changes.
